packages/physics-lite/src/HitResult.ts

```diff
@@ -0,0 +1,16 @@
+import { Vector3 } from "@oasis-engine/math";
+
+/**
+ * Structure used to get information back from a raycast or a sweep.
+ * @internal
+ */
+export class HitResult {
```

Comment:
Rename to  `LiteHitResult` and `LiteTransform` and `LiteUpdateFlag`

Reply:
done